docs/docs/features/argument-parsing/flags.mdx

```diff
@@ -151,6 +151,8 @@ import DefaultFlagCode from "./examples/default-flag.txt";
 
 A flag can be variadic when the type it represents is an array of values. In this case, the flag can be specified multiple times and each value is then parsed individually and added to a single array. If the type of a flag is an array it must be set as variadic.
 
+If the `variadic` config property is set to a string, Stricli will use that as a separator and split each input string. This is useful for cases where the input string is a single value that contains multiple values separated by a specific character (like a comma).
```

Comment:
Should we add an example for this as well? Could move this paragraph below the code example and add another example after that.

Reply:
I agree that it would be good to have a working example, but I'm waiting for #60 to write any more interactive examples.